src/StorageMover/Az.StorageMover.psm1

```diff
@@ -33,7 +33,7 @@
   }
 
   if(-not $accountsModule) {
-    Write-Error "`nThis module requires $accountsName version 2.7.5 or greater. For installation instructions, please see: https://docs.microsoft.com/powershell/azure/install-az-ps" -ErrorAction Stop
+    Write-Error "`nThis module requires $accountsName version 2.7.5 or greater. For installation instructions, please see: https://learn.microsoft.com/powershell/azure/install-az-ps" -ErrorAction Stop
```

Comment:
As the StorageMover code is auto-generated, is the autorest tool already updated @dolauli ?
So we need to do any update for https://github.com/Azure/azure-powershell/tree/generation/src/StorageMover?

Reply:
We will update generation branch and AutoRest.PowerShell functions both.